xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -1521,7 +1535,7 @@ $Msz MB##
   <tr class="xwiki-livetable-display-filters">
     #foreach ($column in $columns)
       #set ($columnProperties = $columnsProperties.get($column))
-      ## Note: Even though the column is hidden, we still need a (potentially hidden) TD to generate valid HTML
+      ## Note: Even though the column is hidden, we still need a (pottentially hidden) TD to generate valid HTML
```

Comment:
```suggestion
      ## Note: Even though the column is hidden, we still need a (potentially hidden) TD to generate valid HTML
```

This weird change seems to introduce a typo.

Reply:
I'm pretty sure this is fixing a typo: https://dictionary.cambridge.org/dictionary/english/potentially
I figured that it was okay to leave it in this PR since this is not the only change for this file.

Reply:
Your change replaces "potentially" by "pottentially" which introduces a typo unless I misunderstood something. The change I'm suggesting fixes it again. Fixing a typo would be okay I think, but my point here is that it isn't fixing but introducing one.

Reply:
Sorry, I misread the changes